design.md

```diff
@@ -1,13 +1,21 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+The target is the same target as the original picobot syntax. Our syntax is equally able to be described in one document. 
 
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
+We chose this design because we like the existing picobot syntax but think that there are a few improvements that could be made. We didnt like that you have to specify the fact that you dont care about a direction (with a *). We think that you should just be able to not care. We also thought that explicitly specifying that you stay in the same state is unecessary and could be optional.
+
+We also noticed that the specification of NEWS is very weird and decided that we should not care what order the directions are given at all. Our syntax allows for people to specify the direction and the state of the direction that they are looking for instead.
```

Comment:
I'm not convinced there's a case where specifying `****` would actually be necessary (or beneficial for that matter).